test/400_contributeAndCallback.js

```diff
@@ -574,7 +574,7 @@ contract('Fullchain', async (accounts) => {
                     authorization.taskid, // task      (authorization)
```

Comment:
When do you test `(deal.callback == address(0) && _resultsCallback.length == 0)` path?

Reply:
acutally we did not test it for now 
![image](https://github.com/iExecBlockchainComputing/PoCo/assets/100280020/e607de95-a825-4c92-b45a-4f1a879ca88a)

Reply:
Yes, we should test both cases :)

Reply:
will check that thanks

Reply:
The first path is test here : 
https://github.com/iExecBlockchainComputing/PoCo/blob/69f35af63dfd36ec3e07641fd365b57e89c48bcd/test/byContract/IexecPoco/06_claim.test.ts#L288

Reply:
so actually we are good ?

Reply:
A bit hacky for the path seen in claim test but why not :thinking: 
I'm not sure to understand the other path I see https://github.com/iExecBlockchainComputing/PoCo/blob/69f35af63dfd36ec3e07641fd365b57e89c48bcd/test/400_contributeAndCallback.js#L481 in this test (?)

Reply:
Thanks for this point, gonna check that